frontend/catalyst/utils/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
-set(CMAKE_CXX_STANDARD 17)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
+cmake_minimum_required(VERSION 3.26)
+
+project(catalyst_frontend)
 
-find_package(Python 3
-    REQUIRED COMPONENTS Interpreter Development.Module
-    OPTIONAL_COMPONENTS Development.SABIModule)
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
Just confirming that this was intended; we're switching to the C++20 standard?

Reply:
Good question, I think I was just updating this file, and in some places we already require C++20, in others only 17, but I figure it's better to unify to the higher number.